docs/CONTRIBUTING.md

```diff
@@ -20,7 +20,7 @@ to check the rendered documentation
 and then push your changes to the repository as a pull request.
 
 If you need help getting started,
-feel free to ask for help on the `#help-contributing` or `#keptn-docs` channels on the [Keptn Slack](https://keptn.sh/community/#slack).
+feel free to ask for help on the `#help-contributing` or `#keptn-docs` channels on the [Keptn Slack](https://cloud-native.slack.com/archives/C017GAX90GM).
```

Comment:
```suggestion
feel free to ask for help on the
[Keptn Slack](https://cloud-native.slack.com/archives/C017GAX90GM)
channel.
```
We just have the #keptn channel on the CNCF slack, not #help-contributing and #keptn-docs